sass/ckeditor-iframe.scss

```diff
@@ -3,32 +3,34 @@
  * Approximate CLF's style in CKEditor instances on the Drupal back-end.
  */
 
-/* Captions */
-.caption > * {
-  background: #f3f3f3;
-  padding: 0.5ex;
-  border: 1px solid #ccc;
-}
+.ck-content {
```

Comment:
Instead of changing this file, make a ckeditor5-iframe.scss file, import this one into that one with the class surrounding the import statement